.eslintrc.json

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": ["react-app", "plugin:prettier/recommended"]
```

Comment:
Good job on the PR. I think the eslint plugin should extend `react` and not `react-app` considering that we are not using react-app but a custom react build. Or is there a reason for this?

Reply:
Thanks for pointing that out, I forgot we are using a custom build for this, let me change it to react.